packages/components/src/item-group/types.ts

```diff
@@ -0,0 +1,68 @@
+type ItemSize = 'small' | 'medium' | 'large';
+
+export interface ItemGroupProps {
+	/**
+	 * Renders borders around each items.
+	 *
+	 * @default false
+	 */
+	isBordered?: boolean;
```

Comment:
I think this just renders a border around the entire item group, no? It doesn't affect the items individually I don't think.

Reply:
As of now, `isBordered` renders a border around the entire group and a border in between each item.

This also means that, is `isBordered` is `true`, changing the value of `isSeparated` doesn't
 cause any visual changes.

https://user-images.githubusercontent.com/1083581/127287469-cec65b43-1379-4f3f-8ec1-a5dd4bbbd6d7.mp4


Should we change the behaviour of `isBordered` so that it only renders a border around the whole group, and let `isSeparated` show/hide borders in between each items?

Reply:
I actually went ahead in [`4c23f28` (#33701)](https://github.com/WordPress/gutenberg/pull/33701/commits/4c23f282dd5bcb6496794debd640cb96ac46a6a5) and changed the behaviour of `isBordered` so that it only affects the border around the item-group (and not the separator in-between items).

Let me know if you find this new behaviour reasonable, or if I should revert back to how it was before

Reply:
> Should we change the behaviour of `isBordered` so that it only renders a border around the whole group, and let `isSeparated` show/hide borders in between each items?

Yeah! That makes sense!